omero/sysadmins/version-requirements.txt

```diff
@@ -502,27 +622,27 @@ OMERO support policies
     * - 3.5
       - from Mar 2013
       - to Oct 2013
-      - Recommended
-      - Deprecated
-      - Dropped
+      - |Recommended|
+      - |Deprecated|
+      - |Dropped|
       - :zerocforum:`Ice 3.5 <announcements/6093-ice-3-5-0-released>`
         :zerocforum:`Ice 3.5.1 <announcements/6283-ice-3-5-1-released>`
     * - 3.6
       - from June 2015
       - to TBA
-      - Supported
-      - Recommended
-      - Supported
+      - |Supported|
+      - |Recommended|
+      - |Supported|
       - :zerocforum:`Ice 3.6.0 <announcements/6631-ice-3-6-0-and-ice-touch-3-6-0-released>`
-        :zerocforum:`Ice 3.6.1 <announcements/45941-ice-3-6-0-and-ice-touch-3-6-1-released>` (broken)
+        :zerocforum:`Ice 3.6.1 <announcements/45941-ice-3-6-0-and-ice-touch-3-6-1-released>` |Broken|
```

Comment:
<img width="259" alt="screen shot 2017-03-03 at 10 07 30" src="https://cloud.githubusercontent.com/assets/1644105/23547126/f810a8f8-fff9-11e6-9b90-36608315735a.png">
A minor point: this was a bit confusing when I first saw it since the symbol looks out of place, but after seeing the source code it makes sense. Not a blocker, if you're happy just leave it as it is.